src/extensions/surveys.tsx

```diff
@@ -246,10 +246,16 @@ export class SurveyManager {
         // Ensure widget container exists if it doesn't
         const shadow = retrieveWidgetShadow(survey, this._posthog)
         const stylesheetContent = style(survey.appearance)
-        const stylesheet = prepareStylesheet(document, stylesheetContent, this._posthog)
+        const WIDGET_STYLE_ID = 'ph-data-style-id'
```

Comment:
Does it make sense to set it as `ph-data-style-id="ph-data-style-id"`? Same key/value? If so, LGTM!

Reply:
yeah no need to have different values, just need a consistent id to check if it's there